deploy/local_single_user_deploy.sh

```diff
@@ -264,21 +264,21 @@ install() {
 deploy() {
     echo -e "${INFO}Deploying ...${NC}"
     install
-    launch_process ${VISOR_BIN} "--config ${CONF_DIR}/visor_conf.json" "visor.log"
+    launch_process ${VISOR_BIN} "--config ${CONF_DIR}/visor_conf.json" "visor.launch.log"
     sleep 2
     num_keepers=${NUM_KEEPERS}
     for (( i=1; i<num_keepers+1; i++ ))
     do
-        launch_process ${KEEPER_BIN} "--config ${CONF_DIR}/keeper_conf_$i.json" "keeper_$i.log"
+        launch_process ${KEEPER_BIN} "--config ${CONF_DIR}/keeper_conf_$i.json" "keeper_$i.launch.log"
```

Comment:
graphers should start before keepers.

Reply:
I will change this into a different PR specific for the Local Deployment script.